core/src/main/scala/ai/h2o/sparkling/backend/utils/H2OContextExtensions.scala

```diff
@@ -113,10 +116,9 @@ trait H2OContextExtensions extends RestCommunication with RestApiUtils with Shel
   protected def getAndVerifyWorkerNodes(conf: H2OConf): Array[NodeDesc] = {
     try {
       lockCloud(conf)
-      val nodes = RestApiUtils.getNodes(conf)
-      verifyWebOpen(nodes, conf)
+      verifyWebOpen(conf)
```

Comment:
Isn't this check necessary only with manually deployed external backend?

Reply:
Usually it is. But I can image really rare case where user would pass `disable_web` to `extraOptions` which then gets propagated to h2o start command

Reply:
ok